.github/workflows/centraldb_angular_backend_check.yaml

```diff
@@ -0,0 +1,34 @@
+name: CentralDashboard-angular Backend check
```

Comment:
Let's rename this to `CentralDashboard-angular Backend Tests` as well as the file to `centraldb_angular_backend_test.yaml`